### PR TITLE
tooltip updates it position when the tooltip text is changed

### PR DESCRIPTION
**Pull request by DanielBretzigheimer**

### CODE REVIEW

File: CefSharp.Wpf/ChromiumWebBrowser.cs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿// Copyright © 2010-2017 The CefSharp Authors. All rights reserved.
+// Copyright © 2010-2017 The CefSharp Authors. All rights reserved.
```
Comment (amaitland) [inline CefSharp.Wpf/ChromiumWebBrowser.cs]: Revert unnecessary formatting changes please.

File: CefSharp.Wpf/ChromiumWebBrowser.cs [hunk 2]
```diff
@@ -1967,6 +1967,10 @@ private void UpdateTooltip(string text)
             }
             else
             {
+                // hide old tooltip before showing the new one to update the position
+                if (toolTip.Content as string != text)
```
Comment (amaitland) [inline CefSharp.Wpf/ChromiumWebBrowser.cs]: Is the if statement even required? Is there a scenario where the same tooltip text is sent through?

File: CefSharp.Wpf/ChromiumWebBrowser.cs (hunk 1 above)
Comment (DanielBretzigheimer) [reply CefSharp.Wpf/ChromiumWebBrowser.cs]: Sorry but my git client does not show me a change at this line (and as you see the two lines are identical). So I am not able to undo this line.

File: CefSharp.Wpf/ChromiumWebBrowser.cs (hunk 2 above)
Comment (DanielBretzigheimer) [reply CefSharp.Wpf/ChromiumWebBrowser.cs]: It is possible to get called when the tooltip was hidden because the mouse was over an area where no tooltip is shown. But in this case the tooltip.IsOpen is already false so no need to set it. But yeah, technically it can be set to IsOpen every time.

File: CefSharp.Wpf/ChromiumWebBrowser.cs (hunk 1 above)
Comment (amaitland) [reply CefSharp.Wpf/ChromiumWebBrowser.cs]: Your editor has likely changed the files codepage.

The lines may look identical, if you did a binary diff they'd be different.

Comment (DanielBretzigheimer) [reply CefSharp.Wpf/ChromiumWebBrowser.cs]: should be fixed now
